channeldb/db.go

```diff
@@ -336,10 +336,10 @@ type DB struct {
 	noRevLogAmtData bool
 }
 
-// Open opens or creates channeldb. Any necessary schemas migrations due
-// to updates will take place as necessary.
-// TODO(bhandras): deprecate this function.
-func Open(dbPath string, modifiers ...OptionModifier) (*DB, error) {
+// OpenTestBoltDB opens or creates channeldb using the boltDB type and enables
```

Comment:
So this function is currently not used, because in all of our testcases we used the default opening options for boltdb:

```
BoltBackendConfig: kvdb.BoltBackendConfig{
			NoFreelistSync:    true,
			AutoCompact:       false,
			AutoCompactMinAge: kvdb.DefaultBoltAutoCompactMinAge,
			DBTimeout:         kvdb.DefaultDBTimeout,
```

Reply:
So I think its best to remove it? @bhandras